terraform/performance/main.tf

```diff
@@ -111,7 +111,7 @@ resource "null_resource" "integration_test" {
       "export PATH=$PATH:/snap/bin:/usr/local/go/bin",
       "echo run integration test",
       "cd ~/amazon-cloudwatch-agent-test",
-      "sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -s -c file:/${local.instance_temp_directory}/${local.cloudwatch_agent_config}",
+      "sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -s -c file:${local.instance_temp_directory}/${local.cloudwatch_agent_config}",
```

Comment:
`${local.instance_temp_directory}` is already has `/` since the value for that variable is `/tmp`. Therefore, removing it to avoid conflict.

Reply:
Agreed. So this never succeeded before, right?

Reply:
No it has not though !!!

Reply:
On linux a double forward slash works fine (at least in bash):
`ls /tmp/` same result as `ls //tmp//`.

Regardless, it makes sense to remove unnecessary slashes.